src/ui/session/content/message_row/bubble.rs

```diff
@@ -213,18 +217,16 @@ impl MessageBubble {
 
         self.remove_css_class("outgoing");
 
-        if let Some(binding) = imp.sender_binding.take() {
-            binding.unwatch();
-        }
```

Comment:
I'm not sure whether this should be removed. It still should unbind if the previous message was a regular one.

Reply:
I'll try out placing a debug print there. If it doesn't print anything, it can be safely removed.

Reply:
Just tried in `@SecretAdTestChannel` and it didn't print anything, while a sponsored message was showing.
So I think this can be safely removed.

Reply:
Unbinding should only happen when list view recycles. Have scrolled up enough? Add a `println` to the other `unbind` line, too - to be sure.

Reply:
Oh, my fault, I tested in a private chat, and there's no sender name in private chats. I added again the unwatch just to be sure and avoid edge cases (three lines of code don't hurt).